dds/DCPS/transport/framework/TransportSendStrategy.cpp

```diff
@@ -1483,10 +1483,10 @@ TransportSendStrategy::direct_send(bool do_relink)
     } else if ((outcome == OUTCOME_PEER_LOST) ||
                (outcome == OUTCOME_SEND_ERROR)) {
       if (outcome == OUTCOME_SEND_ERROR) {
-        ACE_ERROR((LM_WARNING,
-                   ACE_TEXT("(%P|%t) WARNING: Problem detected in ")
-                   ACE_TEXT("send buffer management: %p.\n"),
-                   ACE_TEXT("send_bytes")));
+        VDBG((LM_WARNING,
```

Comment:
Should this be logged at a lower transport debug level?

Reply:
Because this was an unconditional logging I decided to use VDBG just a few lines below, https://github.com/objectcomputing/OpenDDS/blob/500af9520332c16d7682299b42caf1d80282b797/dds/DCPS/transport/framework/TransportSendStrategy.cpp#L1495. I can make it dependent on a level, which one do you prefer?

Reply:
Since it was unconditional before, let's use level 1 here.